### ISSUE TITLE
Add branch-label as an output so it can be passed to other actions and easily commited to

### ISSUE THREAD

**Issue by Tritium-VLK** [peter-evans/create-pull-request#2969]
### Subject of the issue

I'm trying to use this action to create unique branch names, then call another  linting/reporting action, which makes more commits to this branch. 

Here's some example YAML:
```yaml
      - name: Create Pull Request
        id: cpr
        uses: peter-evans/create-pull-request@v6
        with:
          title: ${{ env.PR_TITLE }}
          body: ${{ env.PR_BODY }}
          branch: add-rewards
          branch-suffix: timestamp
          labels: "Gauge Add Reward"
          delete-branch: true

  run_reports:
    needs: run-script-and-create-pr
    uses: "BalancerMaxis/multisig-ops/.github/workflows/run_reports_reusable.yaml@main"
    with:
      pr_number: ${{ needs.run-script-and-create-pr.outputs.pull-request-number }}
      checkout_ref: ${{ needs.run-script-and-create-pr.outputs.pull-request-label }}
```

Pull-request-label is not an available output, and pull_request_ref can't be pushed to as far as I can tell.  The goal is to run with only the GITHUB_TOKEN. 

Will probably resolve by generating the branch string in a separate step, but it'd be nice to be able to grab the end branch-name here. 

### Steps to reproduce

If this issue is describing a possible bug please provide (or link to) your GitHub Actions workflow.

**Comment by peter-evans** [comment]
Hi @Tritium-VLK 

I don't think this would be a problem to add. I'll try and get to it soon.

**Comment by peter-evans** [comment]
I added the output `pull-request-branch`.

Released as `v6.1.0` / `v6`
